campaigns.json

```diff
@@ -1306,8 +1351,7 @@
         ]
       }, 
       "/MinBias_TuneCP5_13TeV-pythia8/RunIISummer19UL16SIM-106X_mcRun2_asymptotic_v3-v1/GEN-SIM": {
-        "SiteWhitelist": [
-          "T1_UK_RAL_Disk", 
```

Comment:
Why do you remove RAL?

Reply:
rucio list-rules cms:/MinBias_TuneCP5_13TeV-pythia8/RunIISummer19UL16SIM-106X_mcRun2_asymptotic_v3-v1/GEN-SIM
5cf544c76f1f457885a1756241576fce  sync_t1_us_fnal_tape  cms:/MinBias_TuneCP5_13TeV-pythia8/RunIISummer19UL16SIM-106X_mcRun2_asymptotic_v3-v1/GEN-SIM#ff324f9b-4cdd-48d5-9e23-49d0b40c9258  OK[14/0/0]              rse=T1_US_FNAL_Tape         1                   2020-09-30 15:13:35